### PR TITLE
ParquetSerializer - A Parquet serializer for PxWeb

### PR DESCRIPTION
**Pull request by trygu**
* Apache Parquet support for PxWeb (https://parquet.apache.org).
* Added integration tests for ParquetSerializer that uses DynamicData to run through all the test-files.
* Added more test file.
* Parquet-serialization uses three datatypes: DateTime for TIMEVALs, double for data and string for other variables.
* Keeps the original TimeValue in a separate column in addition to a parsed DateTime column.
* Handles NPM in separate columns for each content variable. Replaces values in the value-column with a double.NaN.
* Slight fix of the ConsoleTestApp (might be a good idea to re-write it to be a proper console-app?).

### CODE REVIEW

**Comment by trygu** [comment]
@likp: Should be fairly complete by now. I haven't done anything in the PxWeb/PxApi code to support this, as I thought it would be prudent to wait for a check from you guys. Give me a hint as to how you guys would prefer NPM to be handled. It should be fairly straight forward to handle if the only requirement is to add it as a separate column.

**Comment by trygu** [comment]
Ok. Now I promise it's complete.

File: PCAxis.Serializers/Parquet/ParquetBuilder.cs [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+                    foreach (var value in variable.Values)
+                    {
+                        string columnName = $"{variable.Code}_{value.Code}";
+                        dataFields.Add(new DataField(columnName, typeof(double)));
+                        dataFields.Add(new DataField($"{columnName}_symbol", typeof(string))); // Add corresponding symbol column
+                    }
+                }
+                else
+                {
+                    if (variable.IsContentVariable)
```
Comment (likp) [inline PCAxis.Serializers/Parquet/ParquetBuilder.cs:100]: You will probably need to add a value and value_symbol field for PX files that have not defined the content variable.

Comment (trygu) [reply PCAxis.Serializers/Parquet/ParquetBuilder.cs:100]: If I understand you correctly you are referring to PX files without an explicit content/measure? In that case in line 328 I create the content. The tests actually serializes all the PX files, without errors. There are also PX files there with no explicit content as well. If I misunderstood, would you mind providing me with a PX file that won't serialize correctly?

**Review by runejo — APPROVED** [review]

**Pull request merged**